lib/rubygems/basic_specification.rb

```diff
@@ -35,7 +35,14 @@ def initialize
   end
 
   def self.default_specifications_dir
-    File.join(Gem.default_dir, "specifications", "default")
+    Gem.default_specifications_dir
+  end
+
+  class << self
+
+    extend Gem::Deprecate
+    deprecate :default_specifications_dir, "Gem::BasicSpecification.default_specifications_dir", 2020, 2
```

Comment:
2019, 12?

Reply:
I guess `2019, 02` should be more readable, if that is acceptable.